XRTK-Core/Packages/com.xrtk.core/Services/BaseService.cs

```diff
@@ -30,6 +30,9 @@ public abstract class BaseService : Interfaces.IMixedRealityService
         /// <inheritdoc />
         public virtual void Update() { }
 
+        /// <inheritdoc />
+        public virtual void LateUpdate() { }
```

Comment:
Added LateUpdate to base Service implementation and interface. Existing services continue to work as they did since this is an optional Addition. Any issues with this @StephenHodgson @SimonDarksideJ ?

Reply:
This is already part of the Oculus pre pr, which is ready to merge. Just check your changes are in line with that and there should be no merge conflicts. Or better yet, just pull that pr into this one

Reply:
OK, merged with dev, so if you pull you should get fixedupdate and lateupdate included in the service interface